### PR TITLE
G Suite groups

### PR DESCRIPTION
**Pull request by Tabrarian**
Adding screenshots for G Suite groups collection per #123 and TODO doc comments. I've updated the steps per the current flow. Note that G Suite is officially two words (see https://gsuite.google.com/); I think I corrected all the occurrences here.

### CODE REVIEW

**Comment by carise** [comment]
Thanks for this! I think it will be immensely helpful. One small question, could you tweak the image with the client ID and maybe black out that huge number and cover it with text like "a large number" or something? I don't know what's the best way to do this, but just thinking it might be good not to show the real number.

**Comment by Tabrarian** [comment]
Thanks Carise, indeed, I'll get the number obfuscated. I'm also syncing with Michael on Friday to sort out site preview and doc structure process since the GitHub web UI doesn't support certain functions (like moving images to a new folder!). I expect to have this resolved by EOW - thanks for your patience!

**Comment by carise** [comment]
No worries! Thanks once again, I'm excited for the changes!

File: _docs/howto/configure/gsuite-group-collection.md [hunk 1]
```diff
@@ -4,35 +4,44 @@ order: 203
...
-    1.  On the service account row, you should now see a "View Client ID" link. Click it.
-        Copy the "Client ID" value, which will be a large number.
-1.  Enable the service account in your GSuite admin control panel. You must have
+1.  Create a service account to inventory G Suite Google Groups:
+    1.  Go to
+        [Cloud Platform Console Service Accounts](https://console.cloud.google.com/iam-admin/serviceaccounts)
+        and click **Create service account**.
+    1.  On the **Create service account** dialog that appears, set up your service account:
+        1.  Enter a service account name.
+        1.  Select the **Enable G Suite Domain-wide Delegation** checkbox.
```
Comment (blueandgold) [inline _docs/howto/configure/gsuite-group-collection.md]: Do we cover the consent screen somewhere, including screenshot as below?  I know of 2 users that had problems here.

image

Comment (carise) [inline _docs/howto/configure/gsuite-group-collection.md]: @blueandgold Currently we haven't been :(

Comment (Tabrarian) [inline _docs/howto/configure/gsuite-group-collection.md]: That consent screen bit didn't show up for me; can you share any info about how to get there? I'm happy to collaborate on docs as well, if it's too complex for a comment stream.

Comment (blueandgold) [inline _docs/howto/configure/gsuite-group-collection.md]: I think that this consent screen would come put in a new project, when the dwd is enabled the first time.  Can you please verify this?

Comment (Tabrarian) [inline _docs/howto/configure/gsuite-group-collection.md]: Noting belatedly that we closed this one while experimenting with site preview and working in a branch locally after creating it in the web UI. The changes from this PR are published and I'm investigating the consent screen item noted above. I'll follow up on that separately.

**Pull request closed without merging**